common/src/main/java/me/shedaniel/architectury/registry/trade/MerchantOfferAccess.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * This file is part of architectury.
+ * Copyright (C) 2020, 2021 architectury
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package me.shedaniel.architectury.registry.trade;
+
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.trading.MerchantOffer;
+
+public class MerchantOfferAccess {
+    private final MerchantOffer offer;
+    
+    public MerchantOfferAccess(MerchantOffer offer) {
+        this.offer = offer;
+    }
+    
+    public ItemStack getPrimary() {
```

Comment:
these should be getPrimaryCost or exactly same as the mojmap ones

Reply:
```java
public class MerchantOffer {
   public ItemStack baseCostA;
   public ItemStack costB;
   public ItemStack result;
   // ...
}
```

So better `getCostA` and `getCostB` or maybe `getPrimaryPrice` like in `SimpleTrade`?

Reply:
I would use cost instead of price here, but basically yes